spot_rl_experiments/spot_rl/utils/parse_logs_csv.py

```diff
@@ -0,0 +1,126 @@
+import argparse
```

Comment:
This seems to be some utils function that does not need to live in the spot_rl/utils/parse_logs_csv.py. could we move this script to somewhere else in the repo?

Reply:
where else should we move this file to?

Reply:
Any suggestions on where we can move this file or incorporate this function?